modular_citadel/code/modules/research/designs/medical_designs.dm

```diff
@@ -0,0 +1,58 @@
+///////////////////////////////////////////
+/////////Dedibrillators Disks//////////////
+///////////////////////////////////////////
+
+/datum/design/comp_defib
+	name = "compact defibrillator"
+	desc = "A belt-equipped defibrillator that can be rapidly deployed"
```

Comment:
This is way too overpowered of a tool. the defib mounts (which are working now, by the by) provide infinite charge with the bonus of not having it on your back for the cost of not being able to move further than 1 tile away.

a belt defib is going to be mass produced by powergamers and has been intended for CMO use, and ERT/Nukies.